singular_api_client/singular_client.py

```diff
@@ -302,6 +365,22 @@ def _build_reporting_query(cls, start_date, end_date, format, dimensions, metric
         query_dict.update(kwargs)
         return query_dict
 
+    def _build_skan_reporting_query(cls, start_date, end_date, format, dimensions, metrics, discrepancy_metrics,
+                               cohort_metrics, cohort_periods, app, source, display_alignment, time_breakdown,
+                               country_code_format, filters, skadnetwork_date_type, skan_events, **kwargs):
```

Comment:
I see both use cases pass `None` and `[]` as values for all kinds of parameters here - maybe remove them from here and just set the default value is this function?